R/PlotRobustnessSequential.R

```diff
@@ -299,7 +299,7 @@ PlotRobustnessSequential <- function(
       else
         hypothesisSymbol <- '[1]'
 
-      evidenceFor <- gettextf("Evidence for H%s:", hypothesisSymbol, domain="R-jaspGraphs")
+      evidenceFor <- gettextf("Evidence for H%s:", hypothesisSymbol)
       evidenceFor <- fixTranslationForExpression(evidenceFor)
```

Comment:
With the fix I suggested, these lines should become:

```r
   evidenceFor <- fixTranslationForExpression(gettext("Evidence for H%s:"))
   evidenceFor <- sprintf(evidenceFor, hypothesisSymbol)
``` 
because the plot math should now be sprintfed in after making the expression 'safe'.